### ISSUE TITLE
Parallel coordinates

### ISSUE THREAD

**Issue by theiliad** [carbon-design-system/carbon-charts#1161]
Future home of specs

**Comment by mjabbink** [comment]
The scale of the tooltip seems too small compared to the axis and label type sizes. Those also seem to be different type sizes?

**Comment by jeanservaas** [comment]
@mjabbink yeah agree, the tooltip does not look to be the correct scale in those images.

Also just looking at the designs of the zoombar in our storybook vs. what's above in the sketch, it looks like the stroke on your zoom axises is like 3px. There are also two grays being used... it looks like the dark gray 70 for the handles and then a lighter gray for the axis lines... Do we need to grays here or should we just take the current zoombar as is and use it here?

This is the small zoombar that's in our storybook — part of the reason the line is so dark is so it provides a clear contrast against the unselected line when you slide the handles:

![image](https://user-images.githubusercontent.com/43144260/139861921-05291069-99cf-445d-9f25-caed6acd8316.png

The axis numbers also look at bit light. 

And yes, to @theiliad 's point, can we see a view when the axises are zoomed? Right now the chart is changing but the handles are not moving, are you just showing hover states above?

How would the chart work when you zoom multiple variables? you'd just see much fewer lines but all of the lines within the zoom area would be full opacity right @theiliad — and then all of the lines outside of the zoom scope would disappear?

**Comment by theiliad** [comment]
> How would the chart work when you zoom multiple variables? you'd just see much fewer lines but all of the lines within the zoom area would be full opacity right @theiliad — and then all of the lines outside of the zoom scope would disappear?

Yes I think so

**Comment by rafaelamascaro-star** [comment]
Hello everyone, hope you are all well! :) @theiliad @jeanservaas @mjabbink 

Follow attached the graphs changed based on your feedback and also the third one is with the zooming function. 

Here's the sketch file: https://ibm.ent.box.com/folder/149092599296

@jeanservaas I've applied all the colors and the tooltip changes based on your reference. Let me know what you think.

If you have any questions or changes, I'm at your disposal!

Thank you so much and I look forward to hearing from you!

Rafa


![Anatomy_1](https://user-images.githubusercontent.com/64613838/141133250-2fdad660-42e3-4f4e-82f2-9d6ccbdefc6e.png)
![Anatomy_2](https://user-images.githubusercontent.com/64613838/141133272-e776f4d3-14c1-4281-9085-77f4b9ce4cd4.png)
![Anatomy_3](https://user-images.githubusercontent.com/64613838/141133278-4bc85cd2-0844-4ff1-b50c-04205a4f62c3.png)

**Comment by mjabbink** [comment]
@theiliad is this in dev now?

**Comment by theiliad** [comment]
> @theiliad is this in dev now?

Yes. Coming soon

**Comment by vgb1993** [comment]
Looking forward for this one. We would like to represent different dimensions our clients orders. Do you have an ETA?
